apps/ledger-live-mobile/src/screens/FirmwareUpdate/BatteryWarningDrawer.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import QueuedDrawer, { Props as QueuedDrawerProps } from "../../components/QueuedDrawer";
+import { Button, Flex, IconBadge, IconsLegacy, Text } from "@ledgerhq/native-ui";
+import { useTranslation } from "react-i18next";
+import useEnv from "@ledgerhq/live-common/hooks/useEnv";
+import { GetBatteryStatusesActionState } from "@ledgerhq/live-common/deviceSDK/actions/getBatteryStatuses";
+import { Device } from "@ledgerhq/types-devices";
+
+type Props = QueuedDrawerProps & {
+  state: GetBatteryStatusesActionState;
+  device: Device;
+  onRetry: () => void;
+  onQuit: () => void;
+};
+
+const BatteryWarningDrawer: React.FC<Props> = ({ device, state, onRetry, onQuit, ...props }) => {
+  const { t } = useTranslation();
+  const lowBatteryPercentage = useEnv("LOW_BATTERY_PERCENTAGE");
```

Comment:
Maybe inject this value into this component : you're already getting it from the hook with `isBatteryLow` if i'm not wrong ;)

Reply:
`isBatteryLow` is a `boolean` that indicate whether the battery is low and `useEnv("LOW_BATTERY_PERCENTAGE")` represents the percentage at which the battery is considered low.

In this case, we need the percentage to update the i18n text, rather than the boolean.

So, you mean that I can make this number accessible through the hook, in addition to `isBatteryLow` and give it to this component with a props ?

Reply:
Hehe yes i went too fast: i would say what you suggest: returning `lowBatteryPercentage` from the hook, and give it to this component with a prop ;)

Reply:
Done